docs/cookbook/creating-webhooks.md

```diff
@@ -0,0 +1,105 @@
+# Creating webhooks
+
+Webhooks allow applications to communicate with each other in real time by sending HTTP requests when certain events or triggers occur. This can allow your site to automatically send notifications when publishing a document or receive updates from payment portals, for example. This short recipe will demonstrate how to set up webhooks in your Apostrophe project.
+
+## Sending information to a 3rd party webhook
```

Comment:
"Outgoing webhooks" is a common term of art for this. Should be here for SEO reasons and because it's familiar.